app/views/attachment_mailer/new_attachment_notification.html.haml

```diff
@@ -1,23 +1,160 @@
+                                %td.wrapper
+                                  %table.one.columns
+                                    %tr
+                                      %td
+                                      
+                                      %td.expander
+                                %td.wrapper{:class => 'left-border-two'}
+                                  %table.one.columns
+                                    %tr
+                                      %td
+                                      
+                                      %td.expander
+                                %td.wrapper.last{:class => 'no-bottom-padding'}
+                                  %table.ten.columns
+                                    %tr
+                                      %td{:class => 'no-bottom-padding'}
+                                        %img{:src => "http://fakeimg.pl/500x300?text=Preview", :alt => "", :class => 'image-with-border'}/
+                                      %td.expander
+                            %table.row
+                              %tr{:class => 'height-20'}  
+                %table.row
+                  %tr
+                    %td.wrapper.last{:class => 'padding-10'}
+                      %table.twelve.columns
+                        %tr
+                          %td.center{:align => "center", :valign => "top", :class => 'no-bottom-padding'}
+                            %center
+                              %p{ :class => 'paragraph-font-style' }
+                                = t("mailer.new_attachment_email.subheader")
+                %table.row
+                  %tr
+                    %td.center{:align => "center"}
+                      %center
+                        %table.container
+                          %tr
+                            %td.wrapper.last
+                              %table.six.columns
+                                %tr
+                                  %td{:align => "center", :valign => "top"}
+                                    %table.medium-button.round
+                                      %tr
+                                        %td.sub-columns.center{:align => "center", :valign => "top"}
+                                          = link_to proposal_url(@proposal), target: "_blank" do
+                                            = t("mailer.new_attachment_email.cta")
+                          %tr
+                            %td.wrapper.last
+                              %table.six.columns
+                                %tr
+                                  %td{:align => "center", :valign => "top"}
+                                    %table.medium-button.round
+                                      %tr
+                                        %td.sub-columns.center{:align => "center", :valign => "top"}
+                                          = link_to proposal_url(@proposal, anchor: "comments"), target: "_blank" do
```

Comment:
thoughts on updating the partial at `app/views/mail_shared/_proposal_button.html.haml` and using that instead ?